ubuntu/16.04/usr/local/share/container/baseimage-10.sh

```diff
@@ -23,7 +23,7 @@ do_build() {
 }
 
 check_development_start() {
-  if [ "$DEVELOPMENT_MODE" -eq 0 ]; then
+  if [ "$DEVELOPMENT_MODE" == 'true' ]; then
```

Comment:
That would be nice if you could change it as I can see some troubles coming from YAML reading `true` as a boolean, etc...